client/components/Application/Sidebar.tsx

```diff
@@ -4,6 +4,7 @@ import LowerMenu from './LowerMenu'
 import sidebarLogo from '../../assets/ODE_sidebar_logo.svg'
 import Button from '@mui/material/Button'
 import * as store from '@client/store'
+import { t } from 'i18next'
```

Comment:
@guergana 
Just to make sure we're on the same page:
- inside React tree it has to be `useTranslation` hook so a component will re-render on the language change
- outside of React tree (i.e. store/actions) it can be just a direct `t` import from `i18next`

Reply:
> @guergana Just to make sure we're on the same page:
> 
>     * inside React tree it has to be `useTranslation` hook so a component will re-render on the language change
> 
>     * outside of React tree (i.e. store/actions) it can be just a direct `t` import from `i18next`

I wasn't aware of the difference between the two. thanks for the clarification.